README.md

```diff
@@ -99,6 +124,70 @@ e.g. in `app.b3a7772e.js`:
 var e=Object.assign;import{M as t,d as a,u as r,c......
 ```
 
+### Add different banner by file name
+
+filename callbacks are supported since `0.6.0`. Return values of `nul` or `""` means not injected, while string returns are injected.
```

Comment:
Typo: `nul` -> `null` .
If the code is judged by the Falsy value, `undefined` can also be supported (if the user prefers) .